modules/library/main/src/test/java/org/geotools/filter/FilterTest.java

```diff
@@ -215,6 +216,9 @@ protected void setUp() throws SchemaException {
         attributes[13] = calDateTime.getTime();
         attributes[14] = new java.sql.Timestamp(calDateTime.getTimeInMillis());
 
+        // Unicode string
+        attributes[16] = "Bara"+Character.toString((char)241)+"ain";
```

Comment:
@ahuarte47 our project source encoding is UTF-8. This is enforced in maven-compiler-plugin encoding settings in the top-level pom and is documented here:
http://docs.geotools.org/latest/userguide/build/source.html

The unicode would be much easier to read if you wrote the attribute string literal as "Barañain", and used strings such as "*BARAÑ*" in ```testUnicodeLike```. You can enter there literals using a UTF-8-aware editor.